internal/mode/static/nginx/config/http/config.go

```diff
@@ -93,3 +93,21 @@ type ProxySSLVerify struct {
 	TrustedCertificate string
 	Name               string
 }
+
+// httpMatch is an internal representation of an HTTPRouteMatch.
+// This struct is marshaled into a string and stored as a variable in the nginx location block for the route's path.
+// The NJS httpmatches module will look up this variable on the request object and compare the request against the
+// Method, Headers, and QueryParams contained in httpMatch.
+// If the request satisfies the httpMatch, NGINX will redirect the request to the location RedirectPath.
+type RouteMatch struct {
```

Comment:
What's the reason for moving this type here? 
Asking, because it is not used in any other types here (like in Location)